cellmlmanip/units.py

```diff
@@ -120,6 +120,22 @@ def add_custom_unit(self, units_name, unit_attributes):
 
             self._define_pint_unit(units_name, unit_definition)
 
+    def add_preferred_custom_unit_name(self, units_name, unit_attributes):
+        """Set a prefered name for all equivalent custom units. If it does not exist yet, also define a new Pint unit
```

Comment:
Can we follow https://www.python.org/dev/peps/pep-0257/ for docstrings please?

Reply:
Can't see what's wrong with that

Reply:
The first line should be a brief one sentence summary of what the method does (probably just "Set a prefered name for all equivalent custom units." in this case). Then a blank line. Then paragraph-style further details.